pkg/static/login.js

```diff
@@ -164,17 +164,42 @@
     }
 
     function requisites() {
-        function disableLogin(name) {
+        function showBypass(bypass) {
+            if (bypass) {
+                // Selectively show and hide elements
+                show("#login", "#login-details", "#login-override");
+                hide("#get-out-link");
+
+                // Reparent login form to the expander
+                id("login-override-content").appendChild(id("login"));
+
+                // Change the state of the button from primary to warning
+                id("login-button").classList.add("pf-m-warning");
+
+                // Render a "helper text" warning above the log in button
+                document.querySelector("#login .login-actions").insertAdjacentHTML(
+                    "beforebegin",
+                    "<div class='pf-c-helper-text pf-m-warning' id='bypass-warning'>" +
+                    _("Cockpit might not render correctly in your browser") +
+                    "</div>"
+                );
+            } else {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17774-20221024-100226-c3521259-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)